sql/updates/world/master/2025_03_11_00_Darkshore.sql

```diff
@@ -0,0 +1,544 @@
+(34339, 2592, 21, 0),
+(34339, 6289, 5, 0),
+(34339, 5785, 4, 0),
+(34339, 1179, 2, 0),
+
+(34350, 54632, 78, 0), -- Torn Flipper
+(34350, 54632, 10, 0), -- Broken Jaws
+(34350, 56051, 6, 0), -- Half-Digested Roe
+(34350, 56052, 3, 0), -- Quivering Flipper
+(34350, 56053, 0.9, 0), -- Dead Tangled Kelp
+(34350, 62268, 0.8, 0), -- Mangled Flipper
+
+(32996, 2589, 25, 0), -- Linen Cloth
+(32996, 2592, 20, 0), -- Wool Cloth
+(32996, 2287, 4, 0), -- Haunch of Meat
+(32996, 1179, 2, 0), -- Ice Cold Milk
+(32996, 858, 1.2, 0), -- Lesser Healing Potion
+(32996, 929, 0.8, 0), -- Healing potion
+(32996, 2455, 0.6, 0), -- Minor Mana Potion
+(32996, 1205, 0.5, 0), -- Melon juice
+(32996, 3312, 0.4, 0), -- Ceremonial Leather Bracers
+
+(33079, 5523, 52, 0), -- Small Barnacled Clam
+(33079, 2589, 31, 0), -- Linen Cloth
+(33079, 2592, 25, 0), -- Wool Cloth
+(33079, 6289, 6, 0), -- Raw Longjaw Mud Snapper
+(33079, 1179, 3, 0), -- Ice Cold Milk
+(33079, 858, 1.7, 0), -- Lesser Healing Potion
+(33079, 2455, 0.9, 0), -- Minor Mana Potion
+(33079, 6308, 0.6, 0), -- Raw Bristle Whisker Catfish
+(33079, 8959, 0.4, 0), -- Raw Spinefin Halibut
+
+(32868, 5523, 52, 0), -- Small Barnacled Clam
+(32868, 2589, 31, 0), -- Linen Cloth
+(32868, 2592, 25, 0), -- Wool Cloth
+(32868, 1179, 3, 0), -- Ice Cold Milk
+(32868, 858, 1.7, 0), -- Lesser Healing Potion
+(32868, 2455, 0.9, 0), -- Minor Mana Potion
+
+(33980, 5523, 52, 0), -- Small Barnacled Clam
+(33980, 2589, 31, 0), -- Linen Cloth
+(33980, 2592, 25, 0), -- Wool Cloth
+(33980, 1179, 3, 0), -- Ice Cold Milk
+(33980, 858, 1.7, 0), -- Lesser Healing Potion
+(33980, 2455, 0.9, 0), -- Minor Mana Potion
+
+(32928, 55983, 78, 0), -- Inert Elemental Scintilla
+(32928, 7070, 0.8, 0), -- Elemental Water
+(32928, 774, 0.6, 0), -- Malachite
+(32928, 60485, 0.5, 0), -- Crackling Crystals
+(32928, 7079, 0.4, 0), -- Globe of Water
+(32928, 7080, 0.3, 0), -- Essence of Water
+(32928, 2075, 1.8, 0), -- Heavy Mace
+
+(32862, 46318, 52, 1), -- Hellscream's Missive
+(32862, 5523, 52, 0), -- Small Barnacled Clam
+(32862, 2589, 31, 0), -- Linen Cloth
+(32862, 2592, 25, 0), -- Wool Cloth
+(32862, 1179, 3, 0), -- Ice Cold Milk
+(32862, 858, 1.7, 0), -- Lesser Healing Potion
+(32862, 2455, 0.9, 0), -- Minor Mana Potion
+
+(32899, 5523, 52, 0), -- Small Barnacled Clam
+(32899, 2589, 31, 0), -- Linen Cloth
+(32899, 2592, 25, 0), -- Wool Cloth
+(32899, 1179, 3, 0), -- Ice Cold Milk
+(32899, 858, 1.7, 0), -- Lesser Healing Potion
+(32899, 2455, 0.9, 0) -- Minor Mana Potion
+;
+
+
+-- creature_template_difficulty
+
+UPDATE `world`.`creature_template_difficulty` SET `LootID`=32935 WHERE `Entry`=32935 AND `DifficultyID`=0;
+UPDATE `world`.`creature_template_difficulty` SET `LootID`=32935 WHERE `Entry`=32935 AND `DifficultyID`=1;
+
+UPDATE `world`.`creature_template_difficulty` SET `LootID`=33009 WHERE `Entry`=33009 AND `DifficultyID`=0;
+UPDATE `world`.`creature_template_difficulty` SET `LootID`=33009 WHERE `Entry`=33009 AND `DifficultyID`=1;
```

Comment:
if the creature is not spawned in a dungeon you can ignore Difficulty 1 (normal mode).